examples/ssl-factory/src/org/apache/cassandra/security/KubernetesSecretsPEMSslContextFactory.java

```diff
@@ -181,4 +183,4 @@ protected TrustManagerFactory buildTrustManagerFactory() throws SSLException
         String PRIVATE_KEY_PASSWORD_ENV_VAR = "PRIVATE_KEY_PASSWORD_ENV_VAR";
         String TRUSTED_CERTIFICATE_ENV_VAR = "TRUSTED_CERTIFICATE_ENV_VAR";
     }
-}
+}
```

Comment:
can you please check your settings and change this? this happens in every class you touched.